packages/eslint-config-khan/index.js

```diff
@@ -125,10 +125,37 @@ module.exports = {
         /**
          * typescript rules
          */
-        "@typescript-eslint/no-explicit-any": "off",
+
+        // Disallow the `any` type.
+        //
+        // We've disabled this rule because most repos have too many uses of `any`
+        // for this to be useful.  Individual repos can enable this rule if they're
+        // in a position to do so.
+        //
+        // Docs: https://typescript-eslint.io/rules/no-explicit-any
+        "@typescript-eslint/no-explicit-any": OFF,
+
         "@typescript-eslint/no-unused-vars": [
             ERROR,
             {args: "none", varsIgnorePattern: "^_*$"},
         ],
+
+        // Require each enum member value to be explicitly initialized.
+        //
+        // By default enums are assigned integer values starting with 0
+        // and ascending.  This can cause issue when removing or inserting
+        // new members in the enum since this can change their values.
+        // Requiring members to be explicitly initialized prevents this issue.
+        //
+        // Docs: https://typescript-eslint.io/rules/prefer-enum-initializers
+        "@typescript-eslint/prefer-enum-initializers": ERROR,
+
+        // Require all enum members to be literal values.
```

Comment:
I know this has landed and I know this is how the docs talk about this, but to me, "literal values" isn't immediately obvious. I wonder if it'd be useful to use the term "explicit values" somewhere in here?  What we're preventing with this rule is the construction of enums where the value is implicitly assigned by the TS compilation, right?

Reply:
That's a good call out.  `prefer-enum-initializers` requires explicit values.  The other rule requires the explicit values to be literals, e.g. `"hello"`, `5`, etc. as opposed to identifiers or function calls.  I can provide some examples in the docs to clarify, although there are links to the official docs which already have very detailed examples.